src/js/lib/tdconverter.js

```diff
@@ -162,11 +168,12 @@ var TimerDataConverter = execMain(function() {
 				continue;
 			}
 			var name = line[0] + '-' + line[1];
+			var timeMs = Math.round(line[0] == '333mbld' ? ((~~(line[2] / 1000) % 1000000) * 1000) : line[2]);
```

Comment:
For mbld, if we are able to get the number of attempts and the number of success, then it would be better to write them in the comment in the format of "success/attempt".